src/components/SESTooltip/TooltipModalVariant.tsx

```diff
@@ -9,7 +8,8 @@ interface TooltipModalVariantProps extends React.PropsWithChildren {
 }
 
 const TooltipModalVariant: React.FC<TooltipModalVariantProps> = ({ children, openModal, handleCloseModal }) => {
-  const { isLight } = useThemeContext();
+  const theme = useTheme();
```

Comment:
You can use a callback in `sx` props to use the `theme` so you don't need this. Take a look in the [docs](https://github.com/makerdao-ses/ecosystem-dashboard/pull/1569)

Reply:
you are right , done